CONTRIBUTING.md

```diff
@@ -0,0 +1,138 @@
+# Contributing to WHAC19 App
+
+
+## Getting Started & Development
+
+You will need to install and setup Flutter to develop this application. You can read more on how to set up Flutter [here](https://flutter.dev/docs/get-started/install).
+
+**Tip!**: If you are using Visual Studio Code as your IDE (Code Editor), the [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter) is really useful and helpful to perform multiple actions such as:
+
+- the installation of dependencies
+- running the application
+
+### **BETA CHANNEL USAGE**
+
+We are currently using the Flutter beta channel for app development, so once you have the Flutter SDK set-up and working please change to the beta channel using on the command line:
+
+```
+flutter channel beta
+flutter upgrade
+```
+
+### Installing dependencies
+
+To install dependencies, simply run `flutter pub get` in your terminal.
+
+### Developing the app
+
+#### iOS
+
+Initialise an instance of an iOS Simulator or connect physical iOS device, navigate to the cloned repo and run from root:
+
+```sh
+# from terminal
+
+open -a simulator
+
+cd <your-workspace>/wh_covid19_app
+flutter run -d "iPhone"
+```
+
+#### Android
+
+##### Development
+
+If you have a device plugged in just run:
+
+```
+flutter run
+```
+
+Otherwise if you have setup [Android Studio](https://developer.android.com/studio) and/or the Android Software Development Kit (SDK) you can run it over a virtual Android emulator using the Android Debug Bridge (ADB). Just simply run:
+
+```
+flutter run adb
+```
+
+## Contributing as a developer
+
+Check our [issues page](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues?q=is%3Aopen+is%3Aissue+label%3A%22Ready+for+Dev%22) `ready for dev` issues to work on.  Look for issues that are not assigned to somebody.  If you are working on an issue assign yourself to it.  To work on something you must be assigned on the issue. If someone already is, ask if they want help and if they agree add yourself to the assignment. Nobody no in the assignment list should be working on an issue.
+
+Before you submit a PR, please ENSURE that you have run the following commands locally and all the steps pass:
+
+```
+flutter clean
+flutter format .
+flutter analyze
+flutter test
+```
+It's ***strongly*** recommended that you enabled "format on save" in your preferred IDE for Flutter (dart) code files.
+
+When contributing to this repository, please first discuss the change you wish to make via slack, issue, or email or any other method with the owners of this repository before making a change.
+
+**Pro tips**
+
+🔥 If your committing changes to the docs ***ONLY*** add `[skip ci]` to **all** of your commit messages to save build time as we are currently burning build minutes 😬. Also prefix your PR title with `Docs: ` so that its obvious to the project team what the PR is.
```

Comment:
```suggestion
🔥 If you are committing changes to the docs ***ONLY*** add `[skip ci]` to **all** of your commit messages to save build time as we are currently burning build minutes 😬. Also prefix your PR title with `Docs: ` so that its obvious to the project team what the PR is.
```